utils/feltService.ts

```diff
@@ -22,3 +22,12 @@ export function hexToDecimal(hex: string | undefined): string {
 
   return new BN(hex.slice(2), 16).toString(10);
 }
+
+export function gweiToEth(gwei: string): number {
```

Comment:
please use https://github.com/MikeMcl/big.js for managing arbitrarily big amounts